### PR TITLE
Fix trip saving

### LINKED ISSUE #457: Refreshing on saving new monitored trip screen crashes OTP-RR
## Observed behavior (please include a screenshot if possible)

Refreshing the browser before saving a new trip to monitor causes OTP-RR to crash with a `Cannot read properties of undefined (reading 'startTime')` error.
image

## Expected behavior

OTP-RR should fail more gracefully, reverting to the planner menu or retaining the monitoredTrip details after refresh. 

## Steps to reproduce the problem

1. Checkout dev branch of OTP-RR
2. Use FDOT config file.
3. Start OTP
4. Plan a trip, choose an itinerary, and click `Save trip` (You must be logged in)
5. On the `Save new trip screen` refresh the browser without saving. 

SavedTripScreen MonitoredTrips Bug - Reduced

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

Fix #457.

## PR Checklist
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [na] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/user/monitored-trip/saved-trip-screen.js [hunk 1]
```diff
@@ -191,27 +205,37 @@ class SavedTripScreen extends Component {
...
+            )
           }
         </Formik>
       )
     }
 
-    return <AccountPage>{screenContents}</AccountPage>
+    return (
+      <AccountPage>
+        <InvisibleA11yLabel role="status">
```
Comment (amy-corson-ibigroup) [inline lib/components/user/monitored-trip/saved-trip-screen.js:228]: Curious what your thoughts are on combining this status bar with the other existing status bar on the page? I tested with safari + voiceover and this does seem to be working okay, but I am wondering if having multiple status regions could be confusing or play differently on different AT

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good to me, just a question about the live regions

File: lib/components/user/monitored-trip/saved-trip-screen.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/saved-trip-screen.js:228]: For "polite" live regions, it seems fine [per this post as long as the updates aren't caused by the same action](https://gaurav5430.medium.com/quick-accessibility-wins-multiple-aria-live-on-single-action-caveat-b79a6f41e7cc).

**Comment by binh-dam-ibigroup** [comment]
> @binh-dam-ibigroup For me, after reloading a trip to be saved, the days of the week that is runs on are being lost. Is this lost in the query url?
> 
>  trip.refresh.issue.mov

@philip-cline Yes, trying to figure that stuff with #916. There is a race condition or something that makes the UI send the wrong query params to OTP-middleware.

**Comment by philip-cline** [comment]
> > @binh-dam-ibigroup For me, after reloading a trip to be saved, the days of the week that is runs on are being lost. Is this lost in the query url?
> > trip.refresh.issue.mov
> 
> @philip-cline Yes, trying to figure that stuff with #916. There is a race condition or something that makes the UI send the wrong query params to OTP-middleware.

Should this PR be blocked until that is resolved or is this fine?

**Comment by binh-dam-ibigroup** [comment]
This PR fixes the UI crash. #916 fixes the incorrect handling of itinerary checking.

**Review by philip-cline — APPROVED** [review]
Given the query url parsing will be handled in a separate PR this fixes the intended issue. Thanks!

**Review by philip-cline — APPROVED** [review]
Given the query url parsing will be handled in a separate PR this fixes the intended issue. Thanks!

**Pull request merged**